website/docs/compute/local-deployment/local-deployment.md

```diff
@@ -0,0 +1,219 @@
+# Local Deployment
+
+It’s possible to deploy a Compute environment to a local environment using a local Kubernetes cluster.
+
+## Requirements
+
+- [Docker](https://www.docker.com/)
+- [Minikube](https://minikube.sigs.k8s.io/docs/start/)
+- [ngrok](https://ngrok.com/download)
+- [subo](https://github.com/suborbital/subo#installing)
+
+## Steps
+
+### 1. Create a folder for the environment
+
+This is a temporary place where we’ll create and configure our Compute environment:
+
+```bash
+mkdir my-compute; cd my-compute
+```
+
+### 2. Start up our Kubernetes cluster
+
+Kubernetes clusters usually live on the cloud. However, with minikube, we can create a local one to use:
+
+```bash
+minikube start
+```
+
+### 3. Expose our cluster to the internet with ngrok
+
+This command will forward all requests to a randomly-generated URL to `http://localhost:80`
+
+```bash
+ngrok http http://localhost
+```
+
+> **Note:** Jot down that URL generated by ngrok! It’ll look something like [https://84925795ffae.eu.ngrok.io](https://84925795ffae.eu.ngrok.io/)
+
+### 4. Generate your Compute manifests
+
+Next we’ll be using subo to generate our Kubernetes manifest files!
+
+```bash
+subo compute deploy core --dryrun
+```
+
+You will be asked for a domain. Please make sure to enter your domain from ngrok.
+
+These will now live in the `.suborbital/` folder.
+
+### 5. Disable TLS checks in the Compute environment
+
+Open up `.suborbital/scc-controlplane-deployment.yaml` in your editor of choice, and make the following changes:
+
+Under the Builder Container (Line 51):
+
+```yaml
+- name: builder
+          image: suborbital/scc-builder:v0.3.1
+          command: ["builder"]
+
+          ports:
+            - containerPort: 8080
+            - containerPort: 8443
+
+          env:
+            - name: SCC_DOMAIN
+              value: "<YOUR_NGROK_DOMAIN>"
+
+            - name: SCC_TLS_PORT
+              value: "8443"
+
+            - name: SCC_LOG_LEVEL
+              value: "info"
+
+            - name: SCC_CONTROL_PLANE
+              value: "scc-controlplane-service:8081"
+
+          volumeMounts:
+            - mountPath: "/home/scn"
+              name: controlplane-storage
+```
+
+Delete the following line:
+
+`delete containerPort: 8443`
+
+Delete the following key-value pair:
+
+```yaml
+- name: SCC_DOMAIN
+  value: "<YOUR_NGROK_DOMAIN>"
+```
```

Comment:
I would explain *why* we do this -- e.g. "we are disabling the built in TLS certificate provisioning, as ngrok already takes care of this for us".

@ramonh I'd also file an issue on [suborbital/subo](/suborbital/subo/issues/new) to add this as a commandline flag? ` subo compute deploy core --notls` or similar?

Reply:
Ah looks like there sort of is an issue for this already :sweat_smile: 
https://github.com/suborbital/subo/issues/331#issue-1311931808